packages/material-ui/src/AppBar/AppBar.js

```diff
@@ -67,7 +67,7 @@ export const styles = theme => {
 };
 
 const AppBar = React.forwardRef(function AppBar(props, ref) {
-  const { classes, className, color = 'primary', position = 'fixed', ...other } = props;
+  const { classes, className, color = 'primary', position = 'fixed', children, ...other } = props;
```

Comment:
We don't explicit children to minimize bundle size.

Reply:
I figured but to get the same PropTypes I had to be explicit.
I need to make changes upstream to allow control over what to include, currently it only adds PropTypes for props that are used explicitly